lexmapr/pipeline.py

```diff
@@ -966,15 +966,15 @@ def get_gram_chunks(num):
                                 for suff in range(len(suffixes)):
                                     suffixString = suffixes[suff]
                                     sampleRevisedWithSuffix = grm + " " + suffixString
-                                if (sampleRevisedWithSuffix in resource_terms_revised.keys() and not localTrigger):  # Not trigger true is used here -reason
-                                    # resourceId = resourceRevisedTermsDict[sampleRevisedWithSuffix]
-                                    partialMatchedList.append(sampleRevisedWithSuffix)
-                                    status_addendum.append("Suffix Addition- " + suffixString + " to the Input")
-                                    for eachTkn in grmTokens:
-                                        covered_tokens.append(eachTkn)
-                                        if eachTkn in remaining_tokens:
-                                            remaining_tokens.remove(eachTkn)
-                                    localTrigger = True
+                                    if (sampleRevisedWithSuffix in resource_terms_revised.keys() and not localTrigger):  # Not trigger true is used here -reason
```

Comment:
@lexmapr Can you confirm this was an indentation typo? This if-statement is duplicated 5 times for 1-5 component matching, and it is only indented to the same level of the for-loop in the 2-component matching section.

Changing this indentation effected one test: test_full_term_sfx_match.

Reply:
I think it is a typo and I may have introduced it while creating the `pipeline.run()` method. You can see the original structure of this section of code here:

https://github.com/lexmapr/LexMapr/commit/de1238df9da862d0f2d3584b827dbb7e7f3ee58c#diff-43bdea91548ee972633b982f54d60247R1158